### ISSUE TITLE
SPF: LATEST 2020 NPC BOUNDARY

### ISSUE THREAD

**Issue by kriswolves** [datagovsg/datagovsg-datasets#627]
### Please provide the name of the dataset or describe the dataset that you're requesting:

SPF NPC BOUNDARY
### Do you know which agency provides/can provide this data? ?
*( Refer to this link for a full list of agencies: https://data.gov.sg/organization )*
Singapore Police Force MHA

### If there's an issue with a dataset, please provide a description of the problem
No issue just the old SPF NPC boundary website already old

### Please provide the fields expected in this dataset:
*(e.g. month | country | passengers)*
NA

month | country | passengers
------|---------|-------------
2017-09|Singapore|10000

### How frequently would you need this dataset to be updated?
Example: 
- [X] *Test*

- [ ] Annually
- [ ] Monthly
- [ ] Weekly
- [ ] Daily
- [ ] Hourly
- [ ] Real-time

### Can you give a short explanation on how you're planning to use this dataset?
NA not for using.

**Comment by lennardl** [comment]
@kriswolves We have approached the data owners at SPF requesting to update the dataset.

**Comment by kriswolves** [comment]
Thank you. Can you request the highest number of crime rate reported in each NPC yearly? Example this year 2010 highest top 10 crime 1. Orchard 2. Marina bay

2011 highest crime rate
1.marina bay
2.orchard

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by lennardl** [comment]
@kriswolves SPF have replied that the boundary has not changed so the dataset is accurate as of current.

The dataset on crime rate by district is not available

**Comment by kriswolves** [comment]
FYI Juliet division is wrong for woodlands West npc. Now Woodlands Division consist of Woodlands West Woodlands East sembawang yishun North and yishun south
[Image]

[Image]


________________________________

**Comment by kriswolves** [comment]
https://www.police.gov.sg/Contact-Us/Woodlands-Division

FYI the whole spf division link as the old chart already split to another npc

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by lennardl** [comment]
Hi all,
I've informed SPF of the error. Waiting on their response.

**Comment by kriswolves** [comment]
Okay thanks you so much for your work!!! Appreciate it

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by kriswolves** [comment]
Oh can you get the latest information on all the service medal and service ribbon badge on uniform?

________________________________

**Comment by kriswolves** [comment]
Hi did you recieved my previous email?

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by lennardl** [comment]
@kriswolves yes we have requested but since DGS typically displays textual data, I'm not certain how useful it would be?

**Comment by kriswolves** [comment]
Hi did you recieved my file?

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by lennardl** [comment]
@kriswolves Nope did not get any file

**Comment by kriswolves** [comment]
https://drive.google.com/file/d/1Ut77j3Jc-HFNKGmDKxlVMMEPzMHJvqfE/view?usp=drivesdk

This is the error I screenshot on the NPC boundary. Please check.
________________________________

**Comment by lennardl** [comment]
@kriswolves Thanks

**Comment by kriswolves** [comment]
Need wait spf reply again!

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by kriswolves** [comment]
Have the spf replied?

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by lennardl** [comment]
@kriswolves Haven't gotten a response that they have the updated data ready yet.

**Comment by kriswolves** [comment]
Still no response?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by kriswolves** [comment]
Any response?

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by lennardl** [comment]
Yes SPF still hasn't responded on the request

**Comment by kriswolves** [comment]
Can check with them again?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by lennardl** [comment]
@kriswolves I haven't heard back from them will update this issue when I do.

**Comment by kriswolves** [comment]
Hi any updates?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by lennardl** [comment]
@kriswolves I have to apologize but SPF have not responded. It might be better if you brought this up to SPF as they are more likely to take action on direct public feedback.

**Comment by kriswolves** [comment]
I tried before no response too.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by kriswolves** [comment]
How about you sent another email again?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by lennardl** [comment]
@kriswolves I have emailed the relevant people again last Friday when I responded to you on GitHub. Haven't heard back from them yet.

**Comment by kriswolves** [comment]
Okay still no response yet?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by lennardl** [comment]
@kriswolves Sadly no

**Comment by kriswolves** [comment]
Can you give me the contact person in charge? Name or number any form of contacts. I contact them

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by lennardl** [comment]
@kriswolves Sorry I am not able to give you the person's contact for privacy reasons. All public enquiries would have to go to their general channel.

**Comment by kriswolves** [comment]
Okay then please rush them.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by kriswolves** [comment]
Hi is there any update till now?

**Comment by lennardl** [comment]
@kriswolves We have tried to follow up with SPF but there has not been a response. It might work better if you took it up with SPF directly.

**Comment by kriswolves** [comment]
Which department I should contact?  Manpower?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by lennardl** [comment]
@kriswolves It falls under HTX

**Comment by lennardl** [comment]
https://www.htx.gov.sg/
